innerprod/fullysec/dmcfe_test.go

```diff
@@ -25,42 +25,36 @@ import (
 	"github.com/fentec-project/gofe/innerprod/fullysec"
 	"github.com/fentec-project/gofe/sample"
 	"github.com/stretchr/testify/assert"
+	"github.com/pkg/errors"
 )
 
 func Test_DMCFE(t *testing.T) {
-	numClients := 3
+	numClients := 100
 	clients := make([]*fullysec.DMCFEClient, numClients)
-	sumT := data.NewConstantMatrix(2, 2, big.NewInt(0))
 
-	// Generate one matrix per client - the sum of matrices needs to be 0 (modulo order of the group).
-	// In real world setting matrices should be generated using secure multi-party computation. However,
-	// a new scheme is coming which won't require multi-party computation.
-	lim := new(big.Int).Div(bn256.Order, big.NewInt(int64(numClients)))
-	sampler := sample.NewUniform(lim)
+	pubT := make([]data.Matrix, numClients)
+	var err error
```

Comment:
Is this declaration necessary?

Reply:
Ok, I added a colon in `err :=` later on.